servers/src/main/java/tachyon/master/file/meta/InodeTree.java

```diff
@@ -579,21 +591,24 @@ private TraversalResult traverseToInode(String[] pathComponents, boolean persist
      */
     private final Inode mInode;
 
-    private final List<Inode> mPersisted;
+    /**
+     * A list of existing Inodes that are found to be persisted during traversal.
```

Comment:
Your function-level comment is incorrect. It is a list of non-persisted inodes encountered during the traversal (as per the comment I suggested), not "a list of existing inodes that are found to be persisted during traversal".

Also, why use `Inodes` instead of `inodes`, which seems more common in our code base?